packages/communication-react/review/communication-react.api.md

```diff
@@ -379,6 +379,8 @@ export class CallError extends Error {
     timestamp: Date;
 }
 
+// Warning: (ae-incompatible-release-tags) The symbol "errorBarSelector" is marked as @public, but its signature references "DiagnosticsCallFeatureState" which is marked as @beta
```

Comment:
Can these fail the api extractor instead of giving warnings?
(Also can this be resolved in this PR?)

Reply:
tl;dr -> Yes, will do by dropping this API for now :(

Not yet, this is an actual problem in that we're not sure if the backing API is going to be in stable by the time we GA.

Will likely have to drop this API from our library before GA.